### PR TITLE
UHF-1083: Unit map

### PR DESCRIPTION
**Pull request by teroelonen**
Testing instructions are here: https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/108

Also check these PRs:
https://github.com/City-of-Helsinki/drupal-hdbt/pull/103
https://github.com/City-of-Helsinki/drupal-module-helfi-tpr/pull/36
https://github.com/City-of-Helsinki/drupal-helfi/pull/135
https://github.com/City-of-Helsinki/drupal-helfi-kymp/pull/16

### CODE REVIEW

File: conf/cmi/user.role.admin.yml [hunk 1]
```diff
@@ -122,3 +122,12 @@ permissions:
   - 'view all tpr_unit revisions'
   - 'view tpr_service'
   - 'view tpr_unit'
+  - 'access tpr_service overview'
```
Comment (khalima) [inline conf/cmi/user.role.admin.yml:125]: Are these duplicated?

File: conf/cmi/user.role.anonymous.yml [hunk 2]
```diff
@@ -17,3 +17,5 @@ permissions:
   - 'view tpr_unit'
   - 'view tpr_service'
   - 'view tpr_unit'
+  - 'view tpr_service'
```
Comment (khalima) [inline conf/cmi/user.role.anonymous.yml:20]: Duplicated?

File: conf/cmi/user.role.authenticated.yml [hunk 3]
```diff
@@ -18,3 +18,5 @@ permissions:
   - 'view tpr_unit'
   - 'view tpr_service'
   - 'view tpr_unit'
+  - 'view tpr_service'
```
Comment (khalima) [inline conf/cmi/user.role.authenticated.yml:21]: Duplicated?

**Review by khalima — CHANGES_REQUESTED** [review]
Check the duplicated content in user roles.

File: conf/cmi/user.role.admin.yml (hunk 1 above)
Comment (teroelonen) [reply conf/cmi/user.role.admin.yml]: True.dat! Fixed.

File: conf/cmi/user.role.anonymous.yml (hunk 2 above)
Comment (teroelonen) [reply conf/cmi/user.role.anonymous.yml]: Fixed.

File: conf/cmi/user.role.authenticated.yml (hunk 3 above)
Comment (teroelonen) [reply conf/cmi/user.role.authenticated.yml]: Fixed.

**Review by khalima — APPROVED** [review]
Good to go! 👍

**Pull request merged**